servers/quarkus-server/src/main/java/org/projectnessie/server/authz/CompiledAuthorizationRules.java

```diff
@@ -53,7 +57,9 @@ public CompiledAuthorizationRules(QuarkusNessieAuthorizationConfig config) {
    * @return A map of compiled authorization rules
    */
   private Map<String, Script> compileAuthorizationRules() {
-    Map<String, String> rules = new HashMap<>(config.rules());
+
+    // Map<String, String> rules = new HashMap<>(config.rules());
```

Comment:
nit: remove this comment?

Reply:
It's the old code. Just left it so it's maybe easier to revert once the Quarkus issue is fixed.